React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm

```diff
@@ -679,6 +707,81 @@ - (void)removeScrollListener:(NSObject<UIScrollViewDelegate> *)scrollListener
   [self.scrollViewDelegateSplitter removeDelegate:scrollListener];
 }
 
+#pragma mark - Maintain visible content position
+
+- (void)_prepareForMaintainVisibleScrollPosition
+{
+  const auto &props = *std::static_pointer_cast<const ScrollViewProps>(_props);
+  if (!props.maintainVisibleContentPosition) {
+    return;
+  }
+
+  BOOL inverted = false; // TODO: props.inverted
+  BOOL horizontal = _scrollView.contentSize.width > self.frame.size.width;
+  int minIdx = props.maintainVisibleContentPosition.value().minIndexForVisible;
+  for (NSUInteger ii = minIdx; ii < _contentView.subviews.count; ++ii) {
+    // Find the first entirely visible view. This must be done after we update the content offset
+    // or it will tend to grab rows that were made visible by the shift in position
+    UIView *subview = _contentView.subviews[ii];
+    BOOL hasNewView = NO;
+    if (horizontal) {
+      CGFloat leftInset = inverted ? _scrollView.contentInset.right : _scrollView.contentInset.left;
+      CGFloat x = _scrollView.contentOffset.x + leftInset;
+      hasNewView = subview.frame.origin.x > x;
+    } else {
+      CGFloat bottomInset = inverted ? _scrollView.contentInset.top : _scrollView.contentInset.bottom;
+      CGFloat y = self->_scrollView.contentOffset.y + bottomInset;
+      hasNewView = subview.frame.origin.y > y;
+    }
+    if (hasNewView || ii == _contentView.subviews.count - 1) {
+      _prevFirstVisibleFrame = subview.frame;
+      _firstVisibleView = subview;
+      break;
+    }
+  }
+}
+
+- (void)_adjustForMaintainVisibleContentPosition
+{
+  const auto &props = *std::static_pointer_cast<const ScrollViewProps>(_props);
+  if (!props.maintainVisibleContentPosition) {
+    return;
+  }
+
+  std::optional<int> autoscrollThreshold = props.maintainVisibleContentPosition.value().autoscrollToTopThreshold;
+  BOOL inverted = false; // TODO: props.inverted
```

Comment:
Fabric ScrollView doesn't support inverted yet.

Reply:
Yikes, okay that _might_ be a problem for us.

Reply:
Actually it might be that the prop is used in js only and not passed to native, will have to double check that.

Reply:
I verified and this logic doesn't seem to be needed at all so I removed it.